sql/modules/Exempt_funcs.sql.inc

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE pg_temp.test_exempt_funcs (funcname text primary key);

```

Comment:
just a note for the future.

wondering if we want to allow for ongoing diagnostics?  If so we'd want to keep this in a permanent table and delete from it before reload.